tests/test_errors.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# Flask--Resources is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Errors test module."""
+
+import pytest
+from flask import Flask
+
+from flask_resources.resources import CollectionResource, Resource, ResourceConfig
+
+
+# These classes are in the file because they are under test too
+class CustomConfig(ResourceConfig):
+    """Custom resource configuration."""
+
+    item_route = "/custom/<id>"
```

Comment:
is "id" here still okay? since is a reserved keyword... It's working, but should it be named differently?

Reply:
That's the wonderfulness of using `resource_requestctx` as the holder of route arguments: it holds them in a dictionary as string keys i.e. they are not identifiers and therefore are not overriding reserved keywords. Even Flask doesn't give you that.